### PR TITLE
[CUBRIDMAN-16] Add description about JSON in Extending the Relational Data Model section

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CUBRIDMAN-16

### CODE REVIEW

**Review by mhoh3963 — COMMENTED** [review]
please, add description about json in ko/intro.rst.
and add charis as reviewer.

File: en/intro.rst [hunk 1]
```diff
@@ -272,7 +272,10 @@ To address this issue, CUBRID introduces the new concept of the Click Counter th
     *   **MULTISET**: A collection type that allows the duplication of elements. The order of entry is not considered.
     *   **LIST**: A collection type that allows the duplication of elements. Unlike with **SET** and **MULTISET**, the order of entry is maintained.
 
+*   **JSON**
+
+    JavaScript Object Notation (JSON) has become the de facto standard for data-interchanging. JSON, one of the semi-structured data, is not allowed to reside in the relational data model. In CUBRID, However, you can create and query JSON documents using :ref:`SQL functions for JSON<json-fn>`. and you can define a :ref:`JSON data type<json-data-type>` column and stores a JSON document into the column.
```
Comment (charischau) [inline en/intro.rst]: 1.In CUBRID, However, you can create and query JSON documents using :ref:`SQL functions for JSON<json-fn>`.and you can define...

- In CUBRID, However, -->  In CUBRID, however, ('However' change to lower case)

- using :ref:`SQL functions for JSON<json-fn>`. and you can define -->  using :ref:`SQL functions for JSON<json-fn>`. You can... (delete 'and')

Comment (beyondykk9) [reply en/intro.rst]: I'd like to suggest changes as @charischau .

**Review by shparkcubrid — APPROVED** [review]

File: en/intro.rst (hunk 1 above)
Comment (hgryoo) [reply en/intro.rst]: I fixed, thank you.

**Comment by hgryoo** [comment]
I updated the Korean version.

**Review by mhoh3963 — APPROVED** [review]

**Review by charischau — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

**Pull request merged**